src/Kunstmaan/AdminBundle/Entity/Exception.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+namespace Kunstmaan\AdminBundle\Entity;
+
+use DateTimeInterface;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @ORM\Entity(repositoryClass="Kunstmaan\AdminBundle\Repository\ExceptionRepository")
+ * @ORM\Table(
+ *      name="kuma_exception",
+ *      indexes={
+ *          @ORM\Index(name="idx_exception_is_mark", columns={"is_mark"})
+ *      }
+ * )
+ * @UniqueEntity("hash")
+ * @ORM\HasLifecycleCallbacks()
+ */
+class Exception extends AbstractEntity
+{
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=3)
+     */
+    private $code;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    private $url;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", nullable=true)
+     */
+    private $urlReferer;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", nullable=false, unique=true, length=32)
+     */
+    private $hash;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer", nullable=false)
+     */
+    private $used;
```

Comment:
Maybe change this to ```events```? I think this is a better name for it in how you are using it.

Reply:
@Devolicious,  I did not see your message, already changed.